File: Selenium/Activities/Activity3.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FireFoxService
from webdriver_manager.firefox import GeckoDriverManager

service = FireFoxService(GeckoDriverManager().install())
with webdriver.Firefox(service = service) as driver:
    driver.get("https://v1.training-support.net")
    print("Home Page Title is: ",driver.title)
    driver.find_element(By.XPATH,"//a[@id = 'about-link']").click()
    print("New page Title is :",driver.title)
    driver.get("https://v1.training-support.net/selenium/login-form")
    print("Logon page Title is :",driver.title)
    driver.find_element(By.XPATH,"//input[@id = 'username']").send_keys("admin")
    driver.find_element(By.XPATH,"//input[@id = 'password']").send_keys("password")
    driver.find_element(By.XPATH,"//button[text() = 'Log in']").click()
    confirmation_msg = driver.find_element(By.XPATH,"//div[@id = 'action-confirmation']").text
    print("confirmation_msg is :"+confirmation_msg)
    driver.quit()

File: Selenium/Activities/Activity3.java
package activities;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.By;
import org.openqa.selenium.firefox.FirefoxDriver;
import io.github.bonigarcia.wdm.WebDriverManager;
public class Activity3 {
    public static void main(String[] args){
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get("https://v1.training-support.net");
        System.out.println("Home Page Title is :" + driver.getTitle());
        driver.findElement(By.xpath("//a[@id = 'about-link']")).click();
        System.out.println("Title of new page :" +driver.getTitle());

        driver.get("https://v1.training-support.net/selenium/login-form");
        System.out.println("Page Title is :"+ driver.getTitle());
        driver.findElement(By.xpath("//input[@id = 'username']")).sendKeys("admin");
        driver.findElement(By.xpath("//input[@id = 'password']")).sendKeys("password");
        driver.findElement(By.xpath("//button[@type='submit']")).click();
        String comfirmation_msg = driver.findElement(By.xpath("//div[@id = 'action-confirmation']")).getText();
        System.out.println("comfirmation_msg is :" +comfirmation_msg);
        driver.quit();



    }
}
